lib/perl/Genome/WorkflowBuilder/Command.pm

```diff
@@ -90,22 +91,42 @@ sub _get_ptero_execute_method {
 
     my $self = shift;
     my $log_dir = shift;
+    my $ptero_lsf_parameters = $self->_get_ptero_lsf_parameters();
+    $ptero_lsf_parameters->{command} = sprintf(
+        'genome ptero wrapper --command-class %s '
+        .'--method execute --log-directory %s',
+        $self->command, $log_dir);
```

Comment:
This is in line with how the lsf c-api expects commands.  In the shell-command service, we use a different method to launch the job that has a different api, one that expects a list of strings.  We could homogenize these interfaces though, it is worth discussing (perhaps outside this PR).